neo/Network/RPC/RpcServer.cs

```diff
@@ -34,11 +34,13 @@ public sealed class RpcServer : IDisposable
         private readonly NeoSystem system;
         private readonly Wallet wallet;
         private IWebHost host;
+        private Fixed8 maxGasInvoke;
 
-        public RpcServer(NeoSystem system, Wallet wallet = null)
+        public RpcServer(NeoSystem system, Wallet wallet = null, long maxGasInvoke = 0L)
```

Comment:
I suggest that you can use `Fixed8` instead of `long` directly here:

```suggestion
        public RpcServer(NeoSystem system, Wallet wallet = null, Fixed8 maxGasInvoke = default)
```